packages/velaux-ui/src/components/UISchema/index.tsx

```diff
@@ -176,6 +177,32 @@ class UISchema extends Component<Props, State> {
     });
   };
 
+  initFormValues = () => {
```

Comment:
Why do we need to do this?

Reply:
Thanks for your feedback! I realized the issue and have updated the code accordingly. 

Looking forward to your thoughts!